effekt/shared/src/main/scala/effekt/Parser.scala

```diff
@@ -353,7 +353,7 @@ class EffektParsers(positions: Positions) extends EffektLexers(positions) {
     )
 
   lazy val valDef: P[Def] =
-     `val` ~> idDef ~ (`:` ~/> valueType).? ~ (`=` ~/> stmt) ^^ ValDef.apply
+    `val` ~> idDef ~ (`:` ~/> valueType).? ~ (`=` ~/> stmt) ^^ ValDef.apply
```

Comment:
Eventually, we can delete `EffektParsers`. We need to keep `EffektLexers` around for some time, since they are also used for ANSIHighlighting (which could and *should* be changed to use the new lexer, I guess) and the core parsers.